### PR TITLE
Added some badges and some markdown editing on README.md

### PR DESCRIPTION
**Pull request by PatoFlamejanteTV**
Added some shields.io badges and some markdown editing on README.md, mainly editing markdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 ## Modules
 
-- [`core`](https://github.com/BlackMIDIDevs/xsynth/tree/master/core): Handles the core audio rendering functionality.
-- [`clib`](https://github.com/BlackMIDIDevs/xsynth/tree/master/clib): C/C++ bindings for XSynth.
-- [`soundfonts`](https://github.com/BlackMIDIDevs/xsynth/tree/master/soundfonts): A module to parse soundfonts to be used in XSynth.
-- [`realtime`](https://github.com/BlackMIDIDevs/xsynth/tree/master/realtime): The real-time rendering module within XSynth.
-- [`render`](https://github.com/BlackMIDIDevs/xsynth/tree/master/render): A command line utility for rendering MIDIs to audio using XSynth.
-- [`kdmapi`](https://github.com/BlackMIDIDevs/xsynth/tree/master/render): A cdylib wrapper around XSynth to act as a drop in replacement for OmniMIDI/KDMAPI.
+- [`core`](https://github.com/BlackMIDIDevs/xsynth/tree/master/core): **Handles the core audio rendering functionality.**
```
Comment (MyBlackMIDIScore) [inline README.md:12]: I don't think any of these or the title need to be in bold

File: README.md [hunk 2]
```diff
@@ -1,14 +1,20 @@
 <h1 align="center">XSynth</h1>
-<p align="center">A fast Rust-based SoundFont synthesizer designed for high voice counts and low latency.</p>
+<p align="center"><b>A fast Rust-based SoundFont synthesizer designed for high voice counts and low latency.</b></p>
+<p align="center">
+<img alt="GitHub repo size" src="https://img.shields.io/github/repo-size/BlackMIDIDevs/xsynth">
+<img alt="GitHub License" src="https://img.shields.io/github/license/BlackMIDIDevs/xsynth">
+<img alt="GitHub Release" src="https://img.shields.io/github/v/release/BlackMIDIDevs/xsynth">
+<img alt="GitHub Downloads (all assets, all releases)" src="https://img.shields.io/github/downloads/BlackMIDIDevs/xsynth/total">
```
Comment (MyBlackMIDIScore) [inline README.md:7]: Downloads count should be redundant for a library (it doesn't reflect the actual download count from crates.io anyway). The rest of the badges are fine imo

**Review by MyBlackMIDIScore — CHANGES_REQUESTED** [review]

**Comment by PatoFlamejanteTV** [comment]
Ok I changed some things, now I think it's slightly better :D

File: README.md [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 
 ## Modules
 
 - [`core`](https://github.com/BlackMIDIDevs/xsynth/tree/master/core): Handles the core audio rendering functionality.
-- [`clib`](https://github.com/BlackMIDIDevs/xsynth/tree/master/clib): C/C++ bindings for XSynth.
-- [`soundfonts`](https://github.com/BlackMIDIDevs/xsynth/tree/master/soundfonts): A module to parse soundfonts to be used in XSynth.
-- [`realtime`](https://github.com/BlackMIDIDevs/xsynth/tree/master/realtime): The real-time rendering module within XSynth.
-- [`render`](https://github.com/BlackMIDIDevs/xsynth/tree/master/render): A command line utility for rendering MIDIs to audio using XSynth.
-- [`kdmapi`](https://github.com/BlackMIDIDevs/xsynth/tree/master/render): A cdylib wrapper around XSynth to act as a drop in replacement for OmniMIDI/KDMAPI.
+- [`clib`](https://github.com/BlackMIDIDevs/xsynth/tree/master/clib): _C/C++_ bindings for XSynth.
```
Comment (MyBlackMIDIScore) [inline README.md:12]: Could you also remove the italics here? It looks a bit out of place imo

File: README.md [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+- [`soundfonts`](https://github.com/BlackMIDIDevs/xsynth/tree/master/soundfonts): A module to parse soundfonts to be used in XSynth
```
Comment (MyBlackMIDIScore) [inline README.md:13]: Also noticed you removed some of the periods here, please add them back so it looks consistent

**Review by MyBlackMIDIScore — CHANGES_REQUESTED** [review]
Sorry for being so nitpicky lol

**Comment by PatoFlamejanteTV** [comment]
Fixed! :)

**Comment by PatoFlamejanteTV** [comment]
Btw sorry for requesting reviews :(

**Comment by MyBlackMIDIScore** [comment]
> Btw sorry for requesting reviews :(

No worries. Although I noticed you added some commas which break the grammar a bit. Please remove any modifications from the modules section and it should be all good.

**Comment by PatoFlamejanteTV** [comment]
Updated! :)

**Review by PatoFlamejanteTV — COMMENTED** [review]
cool

**Review by MyBlackMIDIScore — APPROVED** [review]

**Pull request merged**